xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -53,6 +53,7 @@
 #include "video/VideoInfoTag.h"
 #include "video/VideoItemArtworkHandler.h"
 #include "video/VideoLibraryQueue.h"
+#include "video/VideoManagerTypes.h"
```

Comment:
Is this new include really needed? You did not make any changes in this cpp file in this PR.

Reply:
left over from before the removal of context menu "Add as version to"